maps/clarion.dmm

```diff
@@ -1454,8 +1454,8 @@
 	},
 /area/station/maintenance/northeast)
 "adf" = (
-/obj/fitness/speedbag/captain,
 /obj/decal/cleanable/dirt,
+/obj/fitness/speedbag/captain,
```

Comment:
looks like automatic changes from a map editor, but you didnt make any other changes

Reply:
No, I got yelled at for removing arbitrary stuff, so now I'm adding them back in

Reply:
No changes to Clarion are currently permitted

Reply:
This doesn't change anything. The speedbag already exists on current Clarion. It was removed in this branch and all I did was put it back in

Reply:
Still a modification.

Reply:
So what, I need to remove again? That'll remove the speedbag from current Clarion, meaning it's still a modification.